4_other_sources.yml

```diff
@@ -28,4 +34,26 @@ targets:
       
   4_other_sources/out/ecosheds_summary.csv:
     command: summarize_ecosheds_data(
-      data_ind = '4_other_sources/out/ecosheds_data.rds.ind', out_file = target_name)
\ No newline at end of file
+      data_ind = '4_other_sources/out/ecosheds_data.rds.ind', out_file = target_name)
+      
+##############################
+## get NorWeST data ##########
+##############################
+
+  norwest_url:
+    command: c(I('https://www.fs.fed.us/rm/boise/AWAE/projects/NorWeST/StreamTemperatureDataSummaries.shtml'))
+    
+  download_base_url:
+    command: c(I('https://www.fs.fed.us/rm/boise/AWAE/projects/NorWeST/downloads/ObservedStreamTemperatureMaps'))
+  
+  dl_filenames:
+    command: get_filenames(url = norwest_url)
+    
+  # this creates a list of files and hashes for the two 
+  # output data files, raw data and raw sites
+  4_other_sources/out/norwest_raw_files.yml:
+    command: do_data_file_tasks(
+      files = dl_filenames,
+      base_url = download_base_url,
+      out_file = target_name,
+      '4_other_sources/src/get_norwest_data.R')
```

Comment:
I agree that your solution for using these data files downstream is (1) sorta clunky and (2) probably about the best we can do, given the remake restriction to one output per command. The only slightly cleaner (?) alternative I can think of would look like:

1. From `do_data_file_tasks()`, return the read-in values of the two indicator files rather than writing a file.

2. Change the above target name to `norwest_raw_files` (an object)

3. Add two more targets below this one:
```yml
  4_other_sources/out/norwest_raw_site_data.rds.ind:
    command: c()
    depends: norwest_raw_files
  4_other_sources/out/norwest_raw_temp_data.feather.ind:
    command: c()
    depends: norwest_raw_files
```
I think that will work? You might not even need the `command: c()` at all, or you might need to beef it up to make it into an acceptable command (e.g., `command: c('')`?)

Reply:
Slight confused by this. If I return an object from `do_data_file_tasks` that contains the hashes from the two output files, won't I need to subset that to create the individual indicator files (something like `command: norwest_raw_files[1]`)?  Or is it okay if it's just equal to both hashes?

Reply:
I'm suggesting that you use `norwest_raw_files` as an indicator object of sorts - its sole job is to guarantee the up-to-dateness of these two indicator files (4_other_sources/out/norwest_raw_site_data.rds.ind and 4_other_sources/out/norwest_raw_temp_data.feather.ind). It contains hashes just because those are a reasonably easy and meaningful thing to return, and maybe someday you'll find it convenient to also be able to inspect `norwest_raw_files` directly (though i'm not counting on that).

Reply:
ah okay -- I was confused because I thought the `command: c()` was doing or creating something, but really we're just creating a dependency on top of the previously-created indicator files. I'm not sure why this didn't click for me, but had to physically do it before it made sense!

Reply:
And, confirmed that your proposed targets and commands work!